pkg/apiserver/rest/apis/v1/types.go

```diff
@@ -382,6 +401,14 @@ type CreateApplicationRequest struct {
 	Component   *CreateComponentRequest `json:"component"`
 }
 
+// CreateConfigRequest is the request body to creates a config
+type CreateConfigRequest struct {
+	Name          string `json:"name" validate:"checkname"`
+	Project       string `json:"project" validate:"checkname"`
```

Comment:
Please remove the validate, it required value is not empty.

Add the alias param.

Reply:
Done.